Makefile

```diff
@@ -17,6 +18,10 @@ define make_docker
 	docker build --no-cache --build-arg SVC_NAME=$(subst docker_,,$(1)) --tag=mainflux/$(subst docker_,,$(1)) -f docker/Dockerfile .
 endef
 
+define make_docker_arm
+	docker build --no-cache --build-arg GOARCH=arm --build-arg GOARM=7 --build-arg SVC_NAME=$(subst docker_arm_,,$(1)) --tag=mainflux/$(subst docker_arm_,,$(1))-arm32v7 -f docker/Dockerfile .
```

Comment:
ARM version is hardcoded (v7). I would advise that never the less we pass the ARGs into Docer containers via ENV variables.

BTW, I need compiling for various architectures, so please keep cross-compiling generic.